src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  padding: 10px;
+
+  @for $month-length from 28 through 31 {
+    & :nth-child(n + 1 + #{$current-length}) {
```

Comment:
didn`t U forget about modifier?

Reply:
And also, what'll happen to your calendar if it's 28-day month?  U didn't use index $month-length